src/Microsoft.OpenApi.OData.Reader/Operation/EdmOperationImportOperationHandler.cs

```diff
@@ -152,5 +153,22 @@ internal static string PathAsString(IEnumerable<string> path)
         {
             return String.Join("/", path);
         }
+
+        /// <inheritdoc/>
+        protected override void SetExternalDocs(OpenApiOperation operation)
+        {
+            if (Context.Settings.ShowExternalDocs)
```

Comment:
```suggestion
            if (Context.Settings.ShowExternalDocs && Context.Model.GetLinkRecord(EdmOperationImport, OperationType, Path) is Link externalDocs)
```

(and remove the other condition)
(and repeat to other places)